test/unit/foreman/renderer/snapshots/ProvisioningTemplate/cloud-init/CloudInit default.snap.txt

```diff
@@ -3,6 +3,17 @@ hostname: snapshothost
 fqdn: snapshothost
 manage_etc_hosts: true
 users: {}
+runcmd:
+- |
+  
```

Comment:
This looks odd - is this valid YAML?

Reply:
Looks like it is - https://yaml.org/spec/1.2/spec.html#id2795688